### PR TITLE
fixed login interface and added method declarations into change passw…

### PR DESCRIPTION
**Pull request by alexsikorski**
…ord page

### CODE REVIEW

File: src/test/java/com/sparta/eng82/interfaces/pages/external/LoginPage.java [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-    LoginPage enterPassword();
+    LoginPage enterPassword(String password);
 
-    LoginPage enterIncorrectEmail();
+    Page login();
 
-    LoginPage enterIncorrectPassword();
+    boolean loginAttempt(String email, String password);
 
-    Page login();
```
Comment (jackingham) [inline src/test/java/com/sparta/eng82/interfaces/pages/external/LoginPage.java:15]: No action to click log in anymore?

Comment (alexsikorski) [reply src/test/java/com/sparta/eng82/interfaces/pages/external/LoginPage.java:15]: login() is clickLogin() effectively, feel free to edit commit in GitHub if you feel it makes sense

Comment (jackingham) [reply src/test/java/com/sparta/eng82/interfaces/pages/external/LoginPage.java:15]: Ah, I missed that sorry. Approved.

**Review by jackingham — APPROVED** [review]
Clarified my misunderstanding, approved.

**Review by GolamCh — APPROVED** [review]
looks good!

**Review by kilo-mike — APPROVED** [review]

**Pull request merged**